crates/sui-json-rpc-api/src/read.rs

```diff
@@ -148,4 +148,14 @@ pub trait ReadApi {
     /// Return the first four bytes of the chain's genesis checkpoint digest.
     #[method(name = "getChainIdentifier")]
     async fn get_chain_identifier(&self) -> RpcResult<String>;
+
+    /// Verify a zklogin signature
+    #[method(name = "verifyZkLoginSignature")]
+    async fn verify_zklogin_signature(
```

Comment:
nit: could you also document what the parameters are? You can probably lift the docs from what you wrote for GraphQL. The only difference is that here you add the doc comments directly on the parameters.

Reply:
agreed that providing better docs for what each of the parameters is expected to be would be nice.